resource-managers/kubernetes/docker/src/main/dockerfiles/spark/Dockerfile

```diff
@@ -49,6 +49,7 @@ COPY sbin /opt/spark/sbin
 COPY kubernetes/dockerfiles/spark/entrypoint.sh /opt/
 COPY kubernetes/dockerfiles/spark/decom.sh /opt/
 COPY examples /opt/spark/examples
+RUN ln -s $(basename $(ls /opt/spark/examples/jars/spark-examples_*.jar)) /opt/spark/examples/jars/spark-examples.jar
```

Comment:
This does `ln -s spark-examples_2.12-3.5.0.jar /opt/spark/examples/jars/spark-examples.jar`, but is `spark-examples_2.12-3.5.0.jar` under current path?

Reply:
No~ The symbolic link file is created at `jar` directory.
```
$ docker run -it --rm spark:latest ls -al /opt/spark/examples/jars  | tail -n6
total 1620
drwxr-xr-x 1 root root    4096 Oct 11 04:37 .
drwxr-xr-x 1 root root    4096 Sep  9 02:08 ..
-rw-r--r-- 1 root root   78803 Sep  9 02:08 scopt_2.12-3.7.1.jar
-rw-r--r-- 1 root root 1564255 Sep  9 02:08 spark-examples_2.12-3.5.0.jar
lrwxrwxrwx 1 root root      29 Oct 11 04:37 spark-examples.jar -> spark-examples_2.12-3.5.0.jar
```

Reply:
The first argument is a link source and the second argument is the location of newly created symbolic link. Since we don't use directory in the symbolic link, this relation is maintained even during copying the whole Spark directory.

Reply:
It's not important for `ln` command. Only the first argument is used as a link location for the newly generate symbolic file.
> is spark-examples_2.12-3.5.0.jar under current path?

Reply:
Hmm, yea, `spark-examples_2.12-3.5.0.jar` is the link source. My question is, does the source exist under the current path of running `ln` command?

> No~ The symbolic link file is created at jar directory.

Does `ln` command run under `jar` directory? I don't see there is command changing to `jar` directory before `ln`.

Do I miss anything here?

Reply:
It seems that we are repeating the same questions. Maybe, are you asking because something is not working, @viirya ? Does it fail in your environment?

Reply:
No, I asked this because I always run `ln` command with link source under current path. I don't know that you can run `ln` with a source in different path. Interesting.

If you test it, then it should be okay.

Reply:
> You can do that in your mac.

Yea, I just tested it locally. It works. 👍